### PR TITLE
Update for Bio-Formats 6.4.0

### PR DESCRIPTION
**Pull request by dgault**
Add release announcement and bump for Bio-Formats 6.3.0
See https://trello.com/c/S2YgJjsm/20-open-pr-against-wwwopenmicroscopyorg

### CODE REVIEW

File: _posts/2020-03-12-bio-formats-6-4-0.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+* added a new ``nobigtiff`` option to ``bfconvert`` to disable automatic switching to BigTiff based upon the 
+  number of pixel bytes (TIFF files larger than 4GB). This may be useful when converting using a compression 
+  codec so that the output file size is less than 4GB
+* fixed a bug in ``xmlvalid`` tool to properly handle lowercasing of file names
+* added new bfGetPlaneAtZCT function to MATLAB toolbox to retrieve a particular plane at a ZCT coordinate 
+  (thanks to Mark Kittisopikul)
+* added a new bfTestInRange helper function to MATLAB toolbox with improved performance and error handling 
+  (thanks to Mark Kittisopikul)
+* fixed a bug when using ``bfconvert`` on multi series files with only a single timepoint, channel or Z slice selected
```
Comment (mtbc) [inline _posts/2020-03-12-bio-formats-6-4-0.md]:
```suggestion
* fixed a bug when using ``bfconvert`` on multi-series files with only a single timepoint, channel or Z slice selected
```

**Comment by sbesson** [comment]
I cannot see the announcement staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/bio-formats/, is this expected?

Otherwise, only other questions from my side are 1- whether the `jxrlib` sentence needs a mention of the relicensing and 2- whether some acknowledgement needs to be added for this work similarly to the previously commissioned readers /cc @chris-allan @melissalinkert

**Comment by dgault** [comment]
Not sure why it is staged, it seems to have been merged ok in https://github.com/snoopycrimecop/www.openmicroscopy.org/commits/gh-pages

There is one line about the JPEGXR change but I can modify it if you have a better wording:
`moved JPEG-XR codec and service from ``formats-gpl`` to ``formats-bsd`` component`

**Comment by sbesson** [comment]
Checked and GitHub was saying "Your site is having problems building: Unable to build page. Please try again later error". Although it seems to be fixed now. Staged at 

https://snoopycrimecop.github.io/www.openmicroscopy.org/2020/03/12/bio-formats-6-4-0.html

File: _posts/2020-03-12-bio-formats-6-4-0.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+File format fixes and improvements:
+
+* Applied Precision CellWorX
+  * added support for multiple Z sections
+* DeltaVision
+  * added and updated objective metadata based on values from softWoRx 7.2.0 (thanks to David Pinto)
+* Hamamatsu NDPI
+  * added support for JPEG-XR compression
+  * added full support for files larger than 4 GB
+  * improved support for a number of additional metadata tags
```
Comment (sbesson) [inline _posts/2020-03-12-bio-formats-6-4-0.md]:
```suggestion
  * improved support for a number of additional metadata tags
  * the Hamamatsu NDPI reader improvements are provided via work from Glencoe Software Inc.
```

**Pull request merged**